pennylane_qiskit/qiskit_device2.py

```diff
@@ -55,7 +55,7 @@
 
 # pylint: disable=protected-access
 @contextmanager
-def qiskit_session(device):
+def qiskit_session(device, **kwargs):
```

Comment:
The docstring must be updated to account for the ``kwargs``. Also, do we mention allowed arguments for these keyword arguments? If they are too much, maybe you could like qiskit's docs for the users.

Reply:
I will update the docstring and also add the qiskit docs for users for reference.